components/monitoring/grafana/README.md

```diff
@@ -18,50 +18,23 @@ First, create the `appstudio-grafana` namespace on each Prometheus or Grafana cl
 $ oc create namespace appstudio-grafana
 ```
 
-### Grafana Datasources
+### Grafana Datasource
```

Comment:
We have this section about datasources, but eventually we need to remove them all. Can we clarify here why they need to be removed?

Reply:
I removed this section from the Grafana Readme and added it to the Monitoring Readme